privateGPT.py

```diff
@@ -31,27 +33,42 @@ def main():
         case _default:
             print(f"Model {model_type} not supported!")
             exit;
-    qa = RetrievalQA.from_chain_type(llm=llm, chain_type="stuff", retriever=retriever, return_source_documents=True)
+    qa = RetrievalQA.from_chain_type(llm=llm, chain_type="stuff", retriever=retriever, return_source_documents= not hide_source)
     # Interactive questions and answers
     while True:
         query = input("\nEnter a query: ")
         if query == "exit":
             break
         
         # Get the answer from the chain
-        res = qa(query)    
-        answer, docs = res['result'], res['source_documents']
+        res = qa(query)
+        answer, docs = res['result'], None if hide_source else res['source_documents']
 
         # Print the result
         print("\n\n> Question:")
         print(query)
         print("\n> Answer:")
         print(answer)
-        
-        # Print the relevant sources used for the answer
-        for document in docs:
-            print("\n> " + document.metadata["source"] + ":")
-            print(document.page_content)
+
+        # Print the relevant sources used for the answer, if source is True
+        if not hide_source and docs:
```

Comment:
```suggestion
        if docs:
```
Nit:
This should be enough, we already check `hide_source` above, if `docs` is `None` we know there are no documents.

Reply:
yep agreed, Opted to remove the ```if``` and assign an empty list instead of ```None``` seems more cleaner and simpler.